asd-aituber/apps/web/lib/emotion-voice-mapping.ts

```
@@ -0,0 +1,241 @@
+
+/**
+ * 便利関数：VOICEVOX用オプション生成
+ */
+export function createEmotionalVoiceOptions(
+  text: string,
+  emotion: Emotion,
+  mode: 'asd' | 'nt' = 'nt',
+  speaker: string | number = '46'
+): Omit<VoicevoxSynthesisOptions, 'serverUrl'> {
+  return EmotionVoiceMapper.createVoicevoxOptions(text, emotion, mode, speaker)
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)